src/Helpers/JWKFetcher.php

```diff
@@ -62,9 +62,44 @@ protected function convertCertToPem($cert)
         return $output;
     }
 
+    /**
+     * Get a JWKS from a URL and format for JWT::decode().
+     *
+     * @param string $jwks_url Full URL to the JWKS.
+     *
+     * @return array
+     */
+    public function getFormatted($jwks_url)
```

Comment:
Just a bit confused by the name. What does it mean to _format_ the JWKS for `JWT::decode()`?

The naming points to an internal transform process, just wondering if the developer needs to understand that detail. Could this be called `getKeysFromJwks` or something similar?

Reply:
`JWT::decode()` is an external library that validates JWT signatures (and a few other things). It takes public key signatures in the form of `$keys[$kid] = $x5c_pem`. 

Good point about the method name, that is a little vague. I don't think we need to say "from JWKS" since we're in the `JWKS` class but I'll rename.